docs/tests/system_checklist.rst

```diff
@@ -104,13 +104,13 @@ Checklist
     Is the processor saving to ES? S3?
 
     * Check Datadog
-      "processor.es.ESCrashStorageRedactedJsonDump.save_raw_and_processed.avg"
+      "processor.es.ESCrashStorageRedactedJsonDump.save_processed_crash.avg"
 
       stage: https://app.datadoghq.com/dash/187676/socorro-stage-perf
       prod: https://app.datadoghq.com/dash/65215/socorro-prod
 
     * Check Datadog
```

Comment:
This is wrong. We need to update the system checklist, but that's a job for another day.